### PR TITLE
[new release] logs-ppx (0.2.0)

### PR DESCRIPTION
**Pull request by ulrikstrid**
CHANGES:

- More hygenic output of logging function `m` -> `logger-function`
- Breaking: Emit `Log` instead of `Logs` since that is the recommendation

### CODE REVIEW

File: packages/logs-ppx/logs-ppx.0.2.0/opam [hunk 1]
```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+synopsis: "PPX to cut down on boilerplate when using Logs"
+description:
+  "PPX to remove the closure when logging with Logs. The PPX will still generate the closure which keeps the nice properties of the Logs library."
+maintainer: ["ulrik.strid@outlook.com"]
+authors: ["Ulrik Strid"]
+license: "BSD-3"
```
Comment (mseri) [inline packages/logs-ppx/logs-ppx.0.2.0/opam:7]:
```suggestion
license: "BSD-3-Clause"
```
Is this the correct one? See https://spdx.org/licenses/

File: packages/logs-ppx/logs-ppx.0.2.0/opam [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+homepage: "https://github.com/ulrikstrid/logs-ppx"
+bug-reports: "https://github.com/ulrikstrid/logs-ppx/issues"
+depends: [
+  "ocaml" {>= "4.8.0"}
+  "dune" {>= "2.6"}
+  "ppxlib"
+  "logs" {with-test}
+]
+build: [
+  ["dune" "subst"] {pinned}
```
[the remark this answers is not in the document]
Comment (mseri) [reply packages/logs-ppx/logs-ppx.0.2.0/opam:17]: This should be called with `dev`

**Comment by mseri** [comment]
Can you upstream these two changes?

File: packages/logs-ppx/logs-ppx.0.2.0/opam (hunk 2 above)
[the remark this answers is not in the document]
Comment (ulrikstrid) [reply packages/logs-ppx/logs-ppx.0.2.0/opam]: This is generated via `dune-project` should this be fixed in dune?

**Comment by ulrikstrid** [comment]
I can fix the license upstream but the `{pinned}` -> `{dev}` is generated by dune, how should I go about that?

**Comment by mseri** [comment]
Should have been fixed in recent versions of dune/dune-release. Did you use the latest dune?

**Comment by ulrikstrid** [comment]
I'm on 2.8.5 of dune, and I should be on 1.4.0 of dune-release, can't validate since I'm using nix and it just says `%%VERSION%%` but 1.4.0 is in nixpkgs

**Comment by ulrikstrid** [comment]
Sorry, after updating to `(lang dune 2.8)` from `(lang dune 2.6)` it generates `{dev}` correctly, I'll update this upstream.

**Comment by mseri** [comment]
Thanks

**Pull request merged**